### DISCUSSION TITLE
Do Event Stream records include Spotlight Vulnerability Findings?

### DISCUSSION THREAD

**Discussion by dkindlund** [CrowdStrike/falconpy#197] [Q&A] [answered]
After reviewing the Streaming API Event Dictionary, I couldn't find any corresponding eventTypes that map to Falcon Spotlight Vulnerability findings -- is that data also included in the Event Stream API?  Or is vulnerability information only available through polling the Falcon Spotlight APIs?

**Comment by jshcodes** [comment, marked as answer]
Hi @dkindlund -

Spotlight findings are __not__ included in Event Streams output.  To access this detail, you will need to poll the Spotlight API as you suggest.  

For other readers, the Event Streams dictionary can be reviewed [here](https://falcon.crowdstrike.com/documentation/62/streaming-api-event-dictionary).
